proto/consensus/msgs.proto

```diff
@@ -0,0 +1,80 @@
+syntax = "proto3";
+package tendermint.proto.consensus;
+
+option go_package = "github.com/tendermint/tendermint/proto/consensus";
+
+import "third_party/proto/gogoproto/gogo.proto";
+import "proto/types/types.proto";
+import "proto/libs/bits/types.proto";
+
+message NewRoundStep {
+  int64  height                   = 1;
+  int32  round                    = 2;
+  uint32 step                     = 3;
+  int64  seconds_since_start_time = 4;
```

Comment:
It'd be nice to add yaml tags whenever a field has underscores

Reply:
we will be removing the usage of gogoproto. We wont be able to provide these tags.

Reply:
hmm ok